final_task/rss_reader/validator.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3.8
+
+import requests
+
+import rss_exceptions as er
+
+
+def check_internet_connection(logger):
+    """
+    Check the internet connection
+    """
+    try:
+        logger.info("Check the Internet connection")
+        requests.get('https://www.google.com/', timeout=1)
+    except requests.exceptions.ConnectionError:
+        raise er.InternetConnectionError("No connection to the Internet.")
+
+
+def check_url_availability(cmd_args, logger):
+    """
+    Check the URL availability
+    """
+    url = cmd_args.source
+    try:
+        response = requests.get(url)
+        logger.info('Check the Internet connection.')
+    except Exception:
+        raise er.UnreachableURLError("No connection to the Internet.")
+    else:
+        logger.info('Connection established.')
+
+
+def check_response_status_code(cmd_args, logger):
+    """
+    Check if the response status code is 200: OK
+    """
+    url = cmd_args.source
+    response = requests.get(url)
+    if response.status_code != 200:
```

Comment:
There is a method .raise_for_status() in response object. You could is it

Reply:
why this option is not suitable?